app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgGraphBuilder.java

```diff
@@ -744,6 +746,15 @@ public Line highValuesLine() {
         return highValuesLine;
     }
 
+    public Line badValuesLine() {
+        Line badValuesLine = new Line (badValues);
```

Comment:
CF: Please remove the space before `(`.

Reply:
Thanks for catching that.  I will never forget to do code formatting again.

Reply:
Uhh, now a lot more changes were introduced which distracts a little from the main changes. Nevertheless, one more consistently formatted file.

Reply:
Are there still issues?